### DISCUSSION TITLE
why  rocksdb.compression not support zstd?

### DISCUSSION THREAD

**Discussion by usrtax** [apache/kvrocks#853] [Q&A] [answered]
I see this in kvrocks.conf write

```
# Specify the compression to use. Only compress level greater
# than 2 to improve performance.
# Accept value: "no", "snappy", "lz4"
# default snappy
rocksdb.compression snappy
```

but rocksdb do support zstd compress

why kvrocks not support this ?

**Comment by git-hulk** [comment]
@usrtax Thanks for your feedback. Kvrocks needs to static link those libraries one by one, you can also file an issue if you want it.

**Comment by git-hulk** [comment, marked as answer]
@torwig has submitted PR #863 to support zstd compression.
